Documentation/conf.py

```diff
@@ -105,7 +105,8 @@
 .. |SCM_ARCHIVE_LINK| replace:: \{l}
 .. |CURRENT_RELEASE| replace:: \{c}
 .. |NEXT_RELEASE| replace:: \{n}
-""".format(s=scm_web, b=branch, a=archive_name, f=archive_filename, l=archive_link, c=current_release, n=next_release)
+.. |CHART_RELEASE| replace:: \{h}
+""".format(s=scm_web, b=branch, a=archive_name, f=archive_filename, l=archive_link, c=current_release, n=next_release, h=release)
```

Comment:
Viewing the docs locally, I noticed that `|CHART_RELEASE|` will point to 1.6.90 when rendering the development version of the docs. Notably, this will also show up at http://docs.cilium.io/en/latest/ .

As long as we provide a helm chart with that version, I think this is fine. Then the question becomes, how do we update this development version of the helm charts and how often?

Do you know how other projects address this problem? It's a bit weird because for the `latest` / development version, we really just want to pull charts from a subdirectory of the cilium repo. Maybe we can set up a github action to generate the helm package and push automatically based upon master or tag push.

Reply:
As far as I know most projects don't package development version of charts. The only project I'm aware of is [rook](https://github.com/rook/rook/blob/master/Documentation/helm-operator.md#installing) and it seem they just have 2 separate helm repositories for `master` and `release` channels. `master` is tagged by git sha and published from the CI pipeline.  It does seem a bit excessive to me since all commits will result in a new chart version. 

Alternative solutions that you might want to consider:

- I think it's not uncommon to recommend to use stable chart but use `latest` image, this could be done via `--set global.tag=latest` for cilium's chart. 

- You could also recommend to pull repository and do `helm install cilium ./install/kubernetes/cilium`, helm can install local chart without repository.

Reply:
It seems reasonable to me if the `latest` version generates something closer to the current instructions (ie download latest source zip from GH) except using your proposal for `helm install cilium ./install/kubernetes/cilium`. We could likely switch on `latest` to generate the helm command and either the "download zip" or "helm repo add" with some slightly more elaborate `conf.py` trickery. What do you think? Would you mind taking a stab at that?

Reply:
@joestringer I have updated docs to conditionally switch to a local chart based on `READTHEDOCS_VERSION` env:

- `stable` will ask to add helm repo and will use `cilium/cilium --version`
- `latest` or empty will ask to pull master branch and will user `./cilium`

Reply:
@joestringer could you please check this PR one more time.